src/style.css

```diff
@@ -0,0 +1,105 @@
+/* #region fonts */
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+  font-weight: normal;
+  font-style: normal;
+}
+
+/* #endregion */
+/* #region page */
```

Comment:
Remove comments. Try it write obvious code that explain all we need without comments)